tools/functional-tester/etcd-runner/main.go

```diff
@@ -43,6 +45,7 @@ func main() {
 	endpointStr := flag.String("endpoints", "localhost:2379", "endpoints of etcd cluster")
```

Comment:
can you change all other flags to use global vars in another pull requests? so it is consistent with your current change. thanks!